Startup.cs

```diff
@@ -32,7 +37,18 @@ public void ConfigureServices(IServiceCollection services)
                 config.UseSqlServer(_config.GetConnectionString("FitnesssTrackerConn"));
             });
 
+            services.AddAutoMapper(Assembly.GetExecutingAssembly());
+
+            services.AddScoped<ILogRepository, LogRepository>();
+            services.AddScoped<IExerciseRepository, ExerciseRepository>();
+            services.AddScoped<IUserRepository, UserRepository>();
+            services.AddScoped<ILogService, LogService>();
```

Comment:
Why did you choose scoped services vs transient?

Reply:
replaced scoped with transient